features/steps/cocoa_steps.rb

```diff
@@ -1,6 +1,7 @@
 def short_scenario_name(scenario)
   unless scenario.end_with? 'Scenario'
-    raise 'All scenario names must end with "Scenario".  There are removed by the test harness and re-added uniformly by the test fixture.'
+    raise 'All scenario names must end with "Scenario".  ' \
+          'There are removed by the test harness and re-added uniformly by the test fixture.'
```

Comment:
🤨

Reply:
Yup, nasty typo that my brain is frequently guilty of making.  I do know the difference between they/there/their/they're really!